configure-hooks-template.ps1

```diff
@@ -0,0 +1,8 @@
+Write-Output "Installing git-secrets global template"
+git secrets --install $env:USERPROFILE/.git-templates/git-secrets
+Write-Output "Registering AWS secret detection globally"
+git secrets --register-aws --global
+
+# Add hooks to all your local repositories.
+Write-Output "Adding hooks to all local repos"
+git config --global init.templateDir $env:USERPROFILE/.git-templates/git-secrets
```

Comment:
You should add:
`$ErrorActionPreference = "Stop"`
to stop this script if an error occurs at any point during execution.